admin-dev/themes/new-theme/scss/components/_form_popover_error.scss

```diff
@@ -1,13 +1,13 @@
 .form-group {
   [class^="popover-error-container"] {
     .popover {
-      max-width: inherit !important;
-      border: inherit !important;
-      border-radius: 5px;
-      box-shadow: 0px 1px 10px lightgrey;
+      max-width: inherit;
+      border: inherit;
+      @include border-radius(5px);
+      box-shadow: 0 1px 10px #d3d3d3;
```

Comment:
Hy not use the color word any more? Maybe a variable instead then?

Reply:
Bootstrap config of stylelint auto transform colors name to hex code, I won't refactor the whole BO 😆